elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -697,26 +717,29 @@ class PfeNavigation extends PFElement {
       const dropdownButtonId = `main-menu__button--${dropdownButton.dataset.machineName}`;
       const dropdownId = `main-menu__dropdown--${dropdownButton.dataset.machineName}`;
       dropdownButton.setAttribute("id", dropdownButtonId);
-      dropdownButton.setAttribute("aria-expanded", "false");
       dropdownButton.parentElement.dataset.buttonId = dropdownButtonId;
 
       // Create wrapper for dropdown and give it appropriate classes and attributes
       const dropdownWrapper = document.createElement("div");
-      // Find other dropdowns by using the .dropdown-content class (such as Search/All Red Hat)
-      // @todo: figure out if this is causing inconsistent toggling of aria-hidden
-      // const otherDropDowns = this.shadowRoot.querySelector(".dropdown-content");
 
       dropdownWrapper.classList.add("pfe-navigation__dropdown-wrapper");
       if (dropdown.classList.contains("pfe-navigation__dropdown--single-column")) {
         dropdownWrapper.classList.add("pfe-navigation__dropdown-wrapper--single-column");
       }
       dropdownWrapper.setAttribute("id", dropdownId);
+      // set aria-hidden to true initially bc the content is hidden on page load
       dropdownWrapper.setAttribute("aria-hidden", "true");
+      mobileMenuToggleContent.setAttribute("aria-hidden", "true");
+      siteSwitcherContent.setAttribute("aria-hidden", "true");
```

Comment:
Same comment with `this._addCloseDropdownAttributes()`

Reply:
@wesruv see above comment